lib/bootstrap.js

```diff
@@ -265,10 +269,11 @@ function bootstrapComponents(site) {
 module.exports = function () {
   const sites = siteService.sites();
 
+  log('info', 'Beginning bootstrap');
```

Comment:
does this need to be logged? if so, a success message would help too

Reply:
Good call, one minute